File: 11_November_Batch/Assignment-1/3_greatestOf3.c
#include<stdio.h>
int greatest(int a,int b,int c){
    if (a>=b&&a>=c)
    {
        return a;
    }
    else if (b>=a&&b>=c)
    {
        return b;
    }
    else if (c>=b&&c>=a)
    {
        return c;
    }
}
int main(){
    int a,b,c,ans;
    scanf("%d%d%d",&a,&b,&c);
    ans=greatest(a,b,c);
    printf("%d",ans);
    return 0;

}

File: 11_November_Batch/Assignment-1/3_greatestOf3.py
def greatest(a, b, c):
    if a >= b and a >= c:
        return a
    elif b >= a and b >= c:
        return b
    else:
        return c

a=int(input("Enter 1st number: "))
b=int(input("Enter 2nd number: "))
c=int(input("Enter 3rd number: "))
ans = greatest(a, b, c)
print(ans)
